donation-tracker/src/modules/campaignDetails.tsx

```diff
@@ -57,29 +65,52 @@ const CampaignDetailsModule = (props: any) => {
   });
 
   return (
-    <Container maxWidth="lg">
-      &nbsp;
-
-      <Typography variant="h2" component="h1">
-        Campaign / Sammlung &quot;{campaignDetails.Title}&quot;
-      </Typography>
-      <Button
-        href={'/campaigns/'}
-        onClick={(event) => handleClickOnLink(event, '/campaigns/')}
-      >
-        &lt;- back to the overview / zurück zur Übersicht
-      </Button>
-      <Typography variant="h5" component="h5">
-        {campaignDetails.ShortCampaignDescription}
-      </Typography>
-      <Typography component="div">
-        {props.children}
-      </Typography>
+    <LayoutModule>
+      <Container maxWidth="lg">
+        &nbsp;
```

Comment:
I saw this quite often. Maybe use a `<Box pt=... />` or so instead.

Reply:
The Container? That is because I did not put it into the Layout yet, could be worth it. but wanted to keep it open in case of need